src/dxvk/dxvk_instance.cpp

```diff
@@ -269,12 +269,18 @@ namespace dxvk {
 
     for (uint32_t i = 0; i < numAdapters; i++) {
       if (filter.testAdapter(deviceProperties[i])) {
-        result.push_back(new DxvkAdapter(m_vki, adapters[i]));
+        auto adapter = new DxvkAdapter(m_vki, adapters[i]);
```

Comment:
`Rc<DxvkAdapter> adatper = new ...` so we can skip the explicit `delete` at the end.

Yeah I know our homebrew refcount thing kind of sucks, but it is what it is.